.github/workflows/ci-e2e-checks.yaml

```diff
@@ -85,7 +85,7 @@ jobs:
       with:
           version: "v0.11.1"
           image: ${{ matrix.kind-node-image }}
-          wait: 360s
+          wait: 600s
```

Comment:
Was this required for tests to complete?

Reply:
It's not required anymore.